backend/src/sms/services/twilio-client.class.ts

```diff
@@ -1,6 +1,6 @@
 import twilio from 'twilio'
```

Comment:
Do we still need to keep this file if we're moving the twilio client to shared?

Reply:
removed, side note: in future to also allow worker to use shared twilio when bandwidth allows (have included this in the comments)